roles/k3s_server/tasks/main.yml

```
@@ -75,13 +75,11 @@
         owner: "{{ ansible_user }}"
         mode: "u=rw,g=,o="
 
-    - name: Add K3s autocomplete to user bashrc
-      become: true
-      become_user: "{{ ansible_user }}"
-      ansible.builtin.command:
-        cmd: "k3s completion bash -i"
-      register: out
-      changed_when: out.rc != 0
+    - name: Add K3s autocomplete to ~{{ ansible_user }}/.bashrc
```

```suggestion
    - name: Add K3s autocomplete to user bashrc
```

Original name avoid lint errors